app/experimenter/experiments/filtersets.py

```diff
@@ -266,7 +266,7 @@ def completed_results_filter(self, queryset, name, value):
         if value:
             return queryset.exclude(
                 results_url=None, results_initial=None, results_lessons_learned=None
-            )
+            ).exclude(results_url="", results_initial="", results_lessons_learned="")
```

Comment:
This would only exclude something if all the fields were `None`, or all the fields were empty strings. It doesn't deal with mixed results, like

```json
{
  "results_url": null,
  "results_initial: "",
}
```

Is that ok? I think you may want to do something more that looks at each field individually and treats both `None` and the empty string as missing values. Maybe something with [`Q` objects](https://docs.djangoproject.com/en/2.2/ref/models/querysets/#q-objects)?

Reply:
@mythmon  so i want something along the lines of... `(results_url=="" OR results_url == null) AND (results_initial=="" OR result_initial==null) AND(results_lesson=="" OR result_lesson==null)`?

Reply:
I think that will work, yeah.